data/help.txt

```diff
@@ -125,3 +125,11 @@ help "trading"
 
 help "friendly disabled"
 	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`
+
+help "fighter fleet logistics"
+	`Fighter fleet logistics has been enabled.  This enables carriers and fighters to support your escorted fleet.  The following logistics are supported.`
+	``
+	`	Fleet refueling if your fighters have fuel and ramscoop.`
```

Comment:
This seems a bit weird, like there will be a list but there is only one?

Reply:
What do you mean by list?  A tanker carrier can also just have ramscoop but that’s less effective than fighters with ramscoop

Reply:
The following logistic**s** are supported, with a space and then that seemed to indicate there would be more than one to me.

Reply:
There will be more than one logistic.  I plan to implement munitions carriers after this pull request is merged.  Munitions carriers are:

* Carriers with lots of ammo storage but not firing the ammo type stored.
* Fighters have ammo storage and not firing the ammo type stored.
* Fighters will distribute ammo from the carrier to the fleet with guns that consume the ammo type to be distributed.

Reply:
Okay, got it, there's more incoming, great! You can resolve this.
Also for the ammo I guess you saw there was a PR merged about that just now.